trezorlib/tx_api.py

```diff
@@ -97,6 +111,12 @@ def get_tx(self, txhash):
             o = t._add_bin_outputs()
             o.amount = int(Decimal(vout['value']) * 100000000)
             o.script_pubkey = binascii.unhexlify(vout['scriptPubKey']['hex'])
+            if self.bip115 and o.script_pubkey[-1] == 0xb4:  # Verify if coin implements replay protection bip115 and script includes checkblockatheight opcode. 0xb4 - is op_code (OP_CHECKBLOCKATHEIGHT)
+                # For exapmle: 76a914b93896509fde251ea0f93cc9f32fa9eb010956b988ac201341e03aaa53e209874363bf85b5cbedf21f12443833f9f721eb500a0000000003943c05b4 bytes we have recieved
```

Comment:
please keep the comment after `if` (and move it to next line), but remove the examples.
instead, please describe the structure, like:
"<original scriptPubKey> <OP_32> <32-byte block hash> <OP_3> <3-byte block height> <OP_CHECKBLOCKATHEIGHT>"
(I'm not sure if I wrote it correctly - If I made a mistake, please write the correct structure :) )

also, I see you removed the length check. In that case I think it's better to extract the "tail" first, and then count forward
```python
tail = o.script_pubkey[-38:]
o.block_hash = tail[1:33]
o.block_height = tail[34:37]
```